src/corelibs/U2Core/src/datatype/msa/MultipleChromatogramAlignmentRow.cpp

```diff
@@ -346,6 +352,22 @@ qint64 MultipleChromatogramAlignmentRowData::getBaseCount(qint64 before) const {
     return MsaRowUtils::getUngappedPosition(gaps, sequence.length(), trimmedRowPos, true);
 }
 
+QPair<DNAChromatogram::ChromatogramTraceAndValue, DNAChromatogram::ChromatogramTraceAndValue>
+                MultipleChromatogramAlignmentRowData::getTwoHighestPeaks(qint64 position) const {
+    const int baseCall = chromatogram.baseCalls[position];
+    QList<DNAChromatogram::ChromatogramTraceAndValue> peaks =
+                                    { { DNAChromatogram::Trace::Trace_A, chromatogram.A[baseCall] },
+                                      { DNAChromatogram::Trace::Trace_C, chromatogram.C[baseCall] },
+                                      { DNAChromatogram::Trace::Trace_G, chromatogram.G[baseCall] },
+                                      { DNAChromatogram::Trace::Trace_T, chromatogram.T[baseCall] } };
+    std::sort(peaks.begin(),
+              peaks.end(),
+        [](const auto& first, const auto& second) {
```

Comment:
At least C++14 has to be turned on on Linux and macOS, I don't want do this change in this very issue, so I rollback `auto`